### PR TITLE
Update components.md

### PR DESCRIPTION
**Pull request by Devops-Ramdas**
I think adding the "for" word in the below sentence makes more sense.

"This document outlines the various components you need to have "for"
a complete and working Kubernetes cluster."

### CODE REVIEW

**Comment by Devops-Ramdas** [comment]
Hi,
I have signed the CNCF Individual Contributor License Agreement. Please review and authorize the PR.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-29498--kubernetes-io-main-staging.netlify.app/docs/concepts/overview/components/

**Comment by jimangel** [comment]
Congrats on your first contribution @Devops-Ramdas! 🎉 
/lgtm
/approve

File: content/en/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ card:
 When you deploy Kubernetes, you get a cluster.
 {{< glossary_definition term_id="cluster" length="all" prepend="A Kubernetes cluster consists of">}}
 
-This document outlines the various components you need to have
+This document outlines the various components you need to have for
```
Comment (kbhawkey) [inline content/en/docs/concepts/overview/components.md:19]: Hi @Devops-Ramdas .
I am not sure this addition is necessary.

This document outlines the various components needed for ...
This document outlines the various components required for a complete and working Kubernetes cluster.

Comment (Devops-Ramdas) [reply content/en/docs/concepts/overview/components.md:19]: Hi @kbhawkey 

although the earlier and current versions look fine, I think this suggestion is more meaningful and technically correct.

```suggestion
This document outlines the various components required for a complete and working Kubernetes cluster.
```

**Pull request merged**